### ISSUE TITLE
After adding review the spinner doesn't disappear

### ISSUE THREAD

**Issue by kapalex** [TechnionYP5777/SmartCity-Accessibility#454]
It shows the newly added review, but the spinner stays

**Comment by ArthurSap** [comment]
Can you tell me what you did so I can recreate the bug? When I added a review the spinner disappeared.

**Comment by kapalex** [comment]
What I did excatly :
1. go to "Kiryon Qiryat Bialik"
2. Click on "Ofer haKiryon"
3. Click on Nearby Places
4. Choose "WeShoes"
5. Click add review
6. Put content and 5 start rating
7. click add

At this point it goes to the previous menu and stays spinning

I did it through the "test" user

**Comment by kapalex** [comment]
Also on android, if it matters

**Comment by ArthurSap** [comment]
Ok so this is a bit long..
It seems to happen while I'm trying to access the user's information (to get the username to be specific) in `reviews.ts` by using `userInformationService.getUserProfile()` (line 80). Next, it goes into `LoginService.java` to the `static UserInfo getUserInfo(String token)` method and here's the problem:
While executing `userInfo = Application.tokenToSession.get(token);` the `user` variable in `userInfo` is NULL !!
I have no idea why is this happening or who is in charge of these classes.
* I also checked with user test
** On Monday (during the meeting) adding a review worked perfectly (No errors thrown, spinner disappeared perfectly, review was added to the server).
